proposals/caffe2-operator-proposal.md

```diff
@@ -0,0 +1,130 @@
+## Motivation
+Caffe2 is a popular machine learning framework which currently does not have an operator/controller for Kubernetes. This proposal is aimed at defining what that operator should look like, and adding it to Kubeflow.
+
+## Goals
+A Kubeflow user should be able to run training using Caffe2 as easily as they can using Tensorflow.  This proposal is centered around a Kubernetes operator for Caffe2. A user should be able to run both single node and distributed training jobs with Caffe2.
```

Comment:
Can you provide some motivation for creating a custom operator? Why are the built in operators unsuitable?

Reply:
Thanks for review, and add the follow motivation:

```
For distributed training, Caffe2 has no parameter server compared with Tensorflow, so it has to use Redis/MPI to find the other nodes to communicate.
```